### DISCUSSION TITLE
[BUG] 面板打开一段时间会卡死

### DISCUSSION THREAD

**Discussion by yuyuko233** [1Panel-dev/1Panel#6167] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

面板在打开使用一段时间后, 不管什么模块都会非常卡, 并且不管处于什么页面, 页面长时间不刷新页面后, 会导致浏览器卡死

### 重现步骤

1. 正常使用一会就能觉得卡顿
2. 在概览界面, 容器列表界面, 日志界面等放置半小时及以上浏览器直接卡死

### 附加信息

![图片](https://github.com/user-attachments/assets/8f1e9665-aea0-48a2-beb1-7b04c347ddc9)
在火狐的进程管理器里也能发现占用一直处于异常, 并且在edge中也会和火狐一样, 长时间就卡死

**Comment by wanghe-fit2cloud** [comment]
感谢反馈！关于页面使用一段时间后浏览器变卡的问题，我们在本地环境中暂时未能复现。
不过，你可以提供一下具体的使用场景、浏览器版本及操作系统信息，以便我们更好地排查问题。
也建议尝试清理浏览器缓存或换个浏览器看看效果。感谢理解！

**Reply by yuyuko233** [reply]
使用场景: 应用商店安装应用, 长时间拉取镜像, 经常查看多个容器日志
浏览器: 火狐 129.0.1 (64 位)
操作系统: Window11 22H2 22621.4037

**Reply by yuyuko233** [reply]
怀疑是那个查看日志的组件, 查看日志的时候首先会页面卡住一段时间, 不管日志的长短, 不过日志如果太长就算关了也有影响, 必须要刷新页面才能恢复正常

**Reply by wanghe-fit2cloud** [reply]
收到，我们尝试在本地环境复现一下这个问题。
